src/adapter/sources.ts

```diff
@@ -217,6 +217,17 @@ export class Source {
     return this._scriptIds;
   }
 
+  /**
+   * Gets a suggested mimetype for the source.
+   */
+  get mimeType(): string | undefined {
```

Comment:
Seems like the name of this getter could convey that it is a guess/suggestion, rather than just relying on comments